src/hooks/useFormState/useFormState.stories.mdx

```diff
@@ -138,10 +142,17 @@ If you're having trouble with yup schema types resolving to `T | undefined` inst
 If the form's initial values are known when it is first rendered, they can simply be passed as the `initialValue` arguments of each `useFormField` call.
 However, let's say you're using a form to edit some object you're loading from the network and you need to wait for that request to resolve before you pre-fill the form.
 
-For this, you can use the `setInitialValue` function on each field object. This changes the current field value, but it also changes the value used to determine if the form `isDirty` and used when `reset`ing the form.
+For this, you can use the `prefill` function on each field object. This changes the current field value, but it also changes the `cleanValue` (used to determine if the form `isDirty` and used when `revert`ing the form).
 
 Here, the submit button is only enabled if the form is dirty, so you can clearly see whether changes have been made to the object from the network that we're editing.
-The reset button also changes the values back to the values we got from the network instead of clearing the form.
+The revert button also changes the values back to the values we got from the network instead of clearing the form.
+
+Note that we have separate buttons here for "Revert" and "Clear". The `revert` function sets each form field to its `cleanValue`, which was changed by the `prefill` function to correspond to the
```

Comment:
Can we simplify this sentance? Maybe remove "which was changed by the `prefill` function to correspond to"?

Reply:
yeah, i had a hard time phrasing this. I'll take another crack at it. Suggestions are welcome on better names for these properties and functions as well.